package.json

```diff
@@ -19,6 +19,7 @@
     "lint": "eslint .",
     "lint:fix": "nr lint --fix",
     "release": "bumpp package.json packages/*/package.json --commit --push --tag && pnpm -r publish --access public",
+    "bench:all": "cross-env CI=true pnpm -r --stream run bench --allowOnly",
```

Comment:
We don't need this, we should only run tests in CI, it doesn't make sense to "just" run benchmarks in CI, we should test benchmark functionality, which is not testable with `vitest bench`.